build.rs

```diff
@@ -1,7 +1,7 @@
 use rustc_version::version_meta;
 use rustc_version::Channel;
 
-#[cfg(feature = "lttng")]
+#[cfg(all(feature = "lttng", target_os = "linux"))]
```

Comment:
Couldn't figure out how to do this with `Cargo.toml` alone. But kinda feels like there should be a way.